services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -902,7 +918,7 @@ protected void handleOfflinePushUpdate(
     routingDataRepository.unSubscribeRoutingDataChange(pushStatus.getKafkaTopic(), this);
 
     if (status.equals(ExecutionStatus.COMPLETED)) {
-      handleCompletedPush(pushStatus);
+      pushStatusCollector.handleServerPushStatusUpdate(pushStatus.getKafkaTopic(), COMPLETED, null);
```

Comment:
but that is filling in different argument right?

Reply:
Never mind, I was thinking we can simply use `status` for the argument in handleServerPushStatusUpdate, but seems that it would simply skip for status other than COMPLETED and ERROR